### ISSUE TITLE
Scanning is working, but no releases are being created

### ISSUE THREAD

**Issue by cleverdevil** [Murodese/pynab#289]
Installed pynab on an Ubuntu 16.04 VM with PostgreSQL as the database. Things seem to be chugging along fine, and I am getting a lot of output in the logs, but the status show no releases being created in the logs, or visible via the API.

I am using supervisor to run all of the services, and the provided tmux-based monitor.

What information can I provide that will help determine the issue?

**Comment by brookesy2** [comment]
More than likely the regex isn't correct/working. Have you tried reloading it, or using different source regex?

**Comment by cleverdevil** [comment]
I just used whatever the default is. I have run `python3 pynab.py regex` several times, but I receive no output from the command, and nothing seems to change. I have one row in the regexes table. Should I have significantly more?

**Comment by cleverdevil** [comment]
I see what's going on. nZEDb no longer has a `master` branch, so the default regex pull is failing. I changed the configuration to `https://raw.githubusercontent.com/nZEDb/nZEDb/master/resources/db/schema/data/10-release_naming_regexes.tsv` instead, and re-ran the `python3 pynab.py regex` command, and it imported a bunch of regexes.

Do I need to wipe my database and start over for it to process all of the data I've already pulled, or will it rescan?
